app/models/concerns/billing_helpers.rb

```diff
@@ -0,0 +1,20 @@
+module BillingHelpers
+  extend ActiveSupport::Concern
+
+  def address
+    country_name = Countries.name_from_alpha2_code(country_code)
+    postal_code_with_city = [postal_code, city].join(' ')
+    [street, postal_code_with_city, country_name].compact.join(', ')
+  end
+
+  def vat_rate
+    if country_code == 'EE'
```

Comment:
Can it be configurable? This prevents other users of software besides us from using it, since country will differ, but logic stays the same.